modules/table/src/main/java/org/apache/ignite/internal/table/distributed/replicator/PartitionReplicaListener.java

```diff
@@ -1330,10 +1335,10 @@ private CompletableFuture<Void> processTxCleanupAction(TxCleanupReplicaRequest r
 
                         cleanupLocally(request.txId(), request.commit(), request.commitTimestamp());
 
-                        return raftClient
-                                .run(txCleanupCmd)
-                                .thenCompose(ignored -> allOffFuturesExceptionIgnored(txReadFutures, request)
-                                        .thenRun(() -> releaseTxLocks(request.txId())));
+                        raftClient.run(txCleanupCmd);
```

Comment:
At this point, while we don't have durable cleanup, it has sense to log an exception from raftClient.run()